core/App/ehsm_core_test.cpp

```diff
@@ -956,11 +956,15 @@ void test_GenerateQuote_and_VerifyQuote()
     printf("============test_GenerateQuote_and_VerifyQuote start==========\n");
     char challenge[32] = "challenge123456";
     char nonce[16] = "nonce123456";
+    char mr_signer[] = "1954701801901551751510511413235234976224826992314220711657250573031213724053";
+    char mr_enclave[] = "13512661971551997612221040105652392282471381951991062211021471876741107354468056";  
```

Comment:
why define it as a char[]? and what's the size of the char[]?

i think the mrenclave should be in the type of " uint8_t [32]" right?

Reply:
Ok, I'll change it as you said.